pkg/option/config.go

```diff
@@ -670,6 +670,9 @@ const (
 	// EnableSRv6 is the name of the option to enable SRv6 encapsulation support
 	EnableSRv6 = "enable-srv6"
 
+	// SRv6ModeName is the name of the option to specify the SRv6 encapsulation mode
+	SRv6ModeName = "srv6-mode"
```

Comment:
In the spirit of calling things what they are, can this actually be "SRv6EncapsulationMode" or "SRv6SRHMode".